src/bento/userDetailViewData.js

```diff
@@ -0,0 +1,33 @@
+import { EDIT } from './adminData';
+
+export const getColumnInfo = (removeRenderer, approvedRenderer, accessType) => [
```

Comment:
@ogwuazors We have to keep custodian file as a simple JSON file in most of the cases. Is it possible to convert getColumnInfo function to Array? If not, we can discuss other options.

Reply:
working on this